labgrid/resource/udev.py

```diff
@@ -301,7 +301,30 @@ class RKUSBLoader(USBResource):
     def filter_match(self, device):
         match = (device.properties.get('ID_VENDOR_ID'), device.properties.get('ID_MODEL_ID'))
 
-        if match not in [("2207", "110a")]:
+        if match[0] != "2207":
+            return False
+
+        model_ids = (
+            "110a",  # RV1108
+            "110b",  # RV1126
+            "300a",  # RK3066
+            "301a",  # RK3036
+            "310b",  # RK3188
+            "310c",  # RK3128
+            "320a",  # RK3288
+            "320b",  # RK322X
+            "320c",  # RK3328
+            "330a",  # RK3368
+            "330c",  # RK3399
+            "330d",  # PX30
+            "330e",  # RK3308
+            "350a",  # RK3568
+            "350b",  # RK3588
+            "350c",  # RK3528
+            "350e",  # RK3576
+        )
+
+        if match[1] not in model_ids:
```

Comment:
Let's stick to the comparison style we have for the other resources:
```suggestion
        if match not in [("2207", "110a"),  # RV1108
                         ("2207", "110b"),  # RV1126
                         ("2207", "300a"),  # RK3066
                         ("2207", "301a"),  # RK3036
                         ("2207", "310b"),  # RK3188
                         ("2207", "310c"),  # RK3128
                         ("2207", "320a"),  # RK3288
                         ("2207", "320b"),  # RK322X
                         ("2207", "320c"),  # RK3328
                         ("2207", "330a"),  # RK3368
                         ("2207", "330c"),  # RK3399
                         ("2207", "330d"),  # PX30
                         ("2207", "330e"),  # RK3308
                         ("2207", "350a"),  # RK3568
                         ("2207", "350b"),  # RK3588
                         ("2207", "350c"),  # RK3528
                         ("2207", "350e"),  # RK3576
                         ]:
```

Reply:
Done